src/production/logic.py

```diff
@@ -174,22 +174,39 @@ def replace_galley_file(article, request, galley, uploaded_file):
 
 
 def save_galley_image(galley, request, uploaded_file, label="Galley Image", fixed=False):
-
+    filename = uploaded_file.name
     if fixed:
```

Comment:
Shouldn't there be a kwarg here called `check_for_existing images` as per https://github.com/BirkbeckCTP/typesetting/pull/204 ?

Reply:
I had the commit but hadn't pushed it :D